Orange/distance/distance.py

```diff
@@ -644,6 +644,36 @@ class PearsonRAbsolute(CorrelationDistance):
     def fit(self, _):
         return PearsonModel(True, self.axis, self.impute)
 
+def _prob_dist(a):
+    # Makes the vector sum to one, as to mimick probability distribution.
+    return a / np.sum(a)
+
+def _bhattacharyya(a, b):
+    # not a real metric, does not obey triangle inequality
+    a = _prob_dist(a)
+    b = _prob_dist(b)
+    if sp.issparse(a):
+        return - np.log(np.sum(np.sqrt(a.multiply(b))))
+    return - np.log(np.sum(np.sqrt(a * b)))
```

Comment:
Sorry for nitpicking. But since you are probably going to change the flie anyway, you can remove space after unary minus (PEP8 is not specific about this, but see the operators in  https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex)